packages/marketplace/src/components/Modals/ImportExport/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import Editor from "react-simple-code-editor";
+import { highlight, languages } from "prismjs/components/prism-core";
+import "prismjs/components/prism-json";
+
+import {
+  exportMarketplace,
+  importMarketplace,
+} from "../../../logic/Utils";
+import Button from "../../Button";
+
+const ImportExportModal = () => {
+  const [code, setCode] = React.useState("");
+
+  // const processCode = () => code.replace(/\n/g, "\\n");
+
+  const exportSettings = () => {
+    // const processedCode = processCode();
+
+    const settings = exportMarketplace();
+    Spicetify.Platform.ClipboardAPI.copy(JSON.stringify(settings));
+    Spicetify.showNotification("Settings copied to clipboard.");
+    Spicetify.PopupModal.hide();
+  };
+
+  const importSettings = () => {
+    const pastedData : string = code;
+    let settings : JSON;
+    // Check if pastedData exists, if not return an error message and exit
+    if (!pastedData) {
+      Spicetify.showNotification("No data pasted");
+      return;
+    }
+    // Check if pastedData is valid JSON, if not return an error message and exit
+    try {
+      settings = JSON.parse(pastedData);
+    } catch (e) {
+      Spicetify.showNotification("Invalid JSON");
+      return;
+    }
+    importMarketplace(settings);
+    Spicetify.PopupModal.hide();
+    location.reload();
```

Comment:
Yeah if we're doing a reload here, we don't need it in the import function itself